src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -239,6 +242,9 @@ export class RoslynLanguageServer {
 
         const server = new RoslynLanguageServer(client, platformInfo, context, telemetryReporter, languageServerEvents);
 
+        server._onAutoInsertFeature = new OnAutoInsertFeature(client);
```

Comment:
could this be moved to the ctor?  So it isn't a nullable field?

Reply:
Duh, yeah. Not sure why that didn't occur to me before. Thanks.